src/redist/targets/Signing.targets

```diff
@@ -183,10 +183,15 @@
 
     <ItemGroup>
       <TemplatesMsiFilesToSign Include="$(Templates50MSIInstallerFile)" />
+    </ItemGroup>
+
+    <ItemGroup Condition="!$(Architecture.StartsWith('arm'))">
       <TemplatesMsiFilesToSign Include="$(Templates31MSIInstallerFile)" />
       <TemplatesMsiFilesToSign Include="$(Templates30MSIInstallerFile)" />
```

Comment:
This is partially related (it also impacts x86/x64), but should we even have 3.0 template files @marcpopMSFT ? 3.0 is considered out-of-support. 5.0 should support anything less than or equal to 5.0

Reply:
This is a valid concern, but it's not something I'd address in this PR. For all I know, 3.0 MSIs should be frozen in VS. I don't know if we keep inserting them at all.

Reply:
Yeah, we're building them, but I don't think we should. We're not inserting these to VS. Just something I noticed.